README.md

```diff
@@ -1,2 +1,54 @@
-# soen342
-A repository for the soen342 project.
+# SOEN342
+A repository for the Craigstutors project.
+
+## Objectives
+Our goal is to create a site to aid students in the best way possible through expert tutelage. Craigstutors is an e-learning platform that allows elementary students to book tutors on a variety of subjects. Our AI-powered personality-matching algorithm rates your child’s compatibility with that of a tutor in order to provide our students with the best possible assistance.
+
+## Core Features
+There are four types of users: Guests, Students, Tutors, and Administrators
```

Comment:
Will we be including parents in the students type?

Reply:
Yes, I believe the idea was to have both parents and students on the same account